tests/test_listish.py

```diff
@@ -159,10 +159,40 @@ def test_get_slice(cls, li):
         assert k is l
 
 
+max_index_size = maxsize - 2 if version_info < (3, 4) else None
```

Comment:
Possibly not worth it because who has lists (/ indexes) this enormous, but the test suite seems to be passing so why not?